test/parallel/test-domain-uncaught-exception.js

```diff
@@ -139,13 +139,13 @@ function test6() {
   const d9 = domain.create();
   const d10 = domain.create();
 
-  d10.on('error', function onErrorInD2Domain(err) {
+  d10.on('error', function onErrorInD2Domain() {
     process.send('errorHandledByDomain');
   });
 
   d9.run(function() {
     d10.run(function() {
-      setTimeout(function onTimeout() {
+      setImmediate(function onSetImmediate() {
```

Comment:
Same here: Use `setTimeout()` with a duration of `1` instead of `setImmediate()`.